scaleout/stackn/requirements.yaml

```diff
@@ -42,10 +42,9 @@ dependencies:
   - name: postgresql-ha
     version: 7.3.0
     repository: https://charts.bitnami.com/bitnami
-    condition: postgresql-ha.enabled
+    condition: postgres-ha.enabled
```

Comment:
Inconsistent naming?

Reply:
thanks for finding this typo!

Reply:
fixed in [commit](https://github.com/scaleoutsystems/charts/pull/72/commits/f6aeb0a37d4ebf63e1b01f9c2d018f71e6318745)